tests/test_newBbox_reading_order.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import os
+from BBOX_sorter import Bbox_sorter
+from tesseract_reader import ImageReader, TesseractReader, TesseractReaderConfig
+from newBbox import newBbox
+
+class test_reading_order_sort(unittest.TestCase):
+    
+    
+    def test_order(self):
+        filenames = [ "test_image1.jpeg",  "test_image2.jpeg", "test_image3.jpeg", "test_image4.jpeg"]
+        expected_points_in_right_order = [
```

Comment:
move the points to a file. Example, for "test_image1.jpeg", [(),(),()]  > "test_image1_points.txt"